components/Cmip6MonthlyChart.vue

```diff
@@ -107,16 +107,30 @@ const buildChart = () => {
         values.push(value)
       })
 
-      traces.push({
-        x: config.years,
-        y: values,
-        mode: 'markers',
-        type: 'scatter',
-        name: config.label,
-        marker: {
-          symbol: config.symbol,
-        },
-      })
+      // Makes chart for sea ice concentration into a line chart
+      if (props.dataKey === 'siconc') {
```

Comment:
I'd prefer to see this made into a prop relating to the functionality instead of inspecting the dataset itself, like, `if (props.mode == 'lines' ) { ... `

Reply:
This has been changed to be the chartType optional property.